### ISSUE TITLE
how to render new views besides the given test/val source views?

### ISSUE THREAD

**Issue by lagrange10** [idiap/GeoNeRF#8]
hello! Thanks for your excellent work!

My project need a series of new images. But neither fine-tune nor eval can generate more new views or a vedio, which is just like pixelNeRF or nerf-pytorch. 

I noticed a file called media in the log file, but I have not ever see a non-empty file called media. Can I get a vedio or more views by this project or modify it?

**Comment by MohammadJohari** [comment]
You need to modify one of the source codes which is responsible for preparing the data. For example, in llff.py we select the validation set with the following code:

val_ids = [idx for idx in range(0, num_viewpoint, 8)]

Which is selecting an image as a validation image from every 8 images in the dataset.

If you want an arbitrary camera pose, you need to create arbitrary camera sequences (c2ws) for your own camera trajectory.

I hope this helps.

**Comment by lagrange10** [comment]
Thanks for your response!

I got an idea thanks to your prompt. I find the c2ws in code `angles = get_angle_wrt_src_cams(c2ws, rays_pts, rays_dir_unit)` and will try to modify it by creating arbitrary camera sequences like `spherify_poses` of the initial NeRF project.

Best regard
Tao
